server/modules/taxonomy/repository/taxonomyRepository.js

```diff
@@ -204,17 +193,17 @@ const findTaxaByPropLike = async (surveyId,
 }
 
 export const findTaxaByCode = async (surveyId,
-                              taxonomyUuid,
-                              filterValue,
-                              draft = false,
-                              client = db) =>
+                                     taxonomyUuid,
+                                     filterValue,
+                                     draft = false,
+                                     client = db) =>
```

Comment:
why not to have those params in one line?

Reply:
there's no reason actually...